server/src/services/listings.ts

```diff
@@ -14,6 +14,11 @@
  * limitations under the License.
  */
 
+import {ListingResponse} from 'interfaces';
+
 import {listingStorage} from '../storage';
 
-export const listingService = {};
+const getAllListings = async (): Promise<ListingResponse[]> =>
+  await listingStorage.getAllListings();
```

Comment:
Ehh you don't need await here also right? Haha
```suggestion
  listingStorage.getAllListings();
```